http/codegen/openapi/v3/builder.go

```diff
@@ -30,6 +30,10 @@ func New(root *expr.RootExpr) *OpenAPI {
 		return nil
 	}
 
+	if !mustGenerateRandomExample(root.API.Meta) {
```

Comment:
I don't understand why this couldn't just re-use the existing `mustGenerate` function? what am I missing :)

Reply:
Oh duh sorry somehow I managed to miss that `mustGenerate` is not about examples 🤦 . Just in-lining the check in `New` makes sense then.

Reply:
No worries, I've pushed the changes.